Packs/Palo_Alto_Networks_WildFire/Integrations/WildFireReports/WildFireReports.py

```diff
@@ -0,0 +1,380 @@
+import demistomock as demisto
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+
+import requests
+import traceback
+from typing import Dict, Any
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+
+
+class NotFoundError(Exception):
+    """ Report or File not found. """
+
+    def __init__(self, *args):  # real signature unknown
+        pass
+
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+    def __init__(self, base_url, verify=True, proxy=False, ok_codes=(), headers=None, auth=None, token=None):
+        super().__init__(base_url, verify, proxy, ok_codes, headers, auth)
+        self.token = token
+
+    def test(self):
```

Comment:
let's do in the test_module something more subtle, e.g getting the report for a hash
https://docs.paloaltonetworks.com/wildfire/u-v/wildfire-api/get-started-with-the-wildfire-api/make-your-first-wildfire-api-call.html#id67bf31b8-9a8f-41e9-a98f-c632571c2255

Reply:
It's even better because we already implement this endpoint